### DISCUSSION TITLE
MiniOS como sistema principal

### DISCUSSION THREAD

**Discussion by AlexR012** [minios-linux/minios-live#30] [Ideas]
Me gustaría saber si habrá la posibilidad de que haya una imagen que no solo sea live y pueda instalarse en un disco duro sin tener el archivo de savechanges

**Comment by FershoUno** [comment]
Hola, cuando instalas MiniOS, no es necesario utilizar "savechanges" ya que todos los cambios seran persistentes, y desde luego se puede instalar en un disco duro :D

**Reply by AlexR012** [reply]
Hola!!, muchas gracias por responder, pero se que los cambios persistentes se guardan en un archivo sin ser necesario el savechanges, es solo que si yo quisiera limpiar o borrar archivos para ahorrar espacio el archivo donde se hacen los cambios persistentes sigue aumentando de tamaño

**Reply by FershoUno** [reply]
Ah estas usando MiniOS sobre Fat32, si deseas evitar esas molestias, puedes instalar MiniOS sobre Ext4, de esa manera no tendrías ningún inconveniente :D

**Reply by AlexR012** [reply]
Muchas gracias!!, instale el MiniOS en un disco duro con el formato ext4 y funcionó, ya los datos no se mantienen en un archivo te agradezco la ayuda ✨

**Reply by FershoUno** [reply]
genial, que bueno que te funciono :smile: :+1:
